base/applications/cmdutils/replace/replace.c

```diff
@@ -437,15 +425,13 @@ INT cmd_replace (LPTSTR param)
                     break;
                 default:
                     invalid_switch(arg[i]);
-                    freep(arg);
-                    return 0;
+                    return 1; /* Error */
                 }
             }
             else
             {
                 invalid_switch(arg[i]);
-                freep(arg);
-                return 0;
+                return 1; /* Error */
```

Comment:
Not sure we need the `/* OK */` and `/* Error */` comments. If you want to have self-documented return values, use the `EXIT_SUCCESS` (== 0) and `EXIT_FAILURE` (== 1) values from stdlib.h